### DISCUSSION TITLE
Welcome to checkedc-llvm-project Discussions!

### DISCUSSION THREAD

**Discussion by arunkumarbhattar** [arunkumarbhattar/CheckCBox_Compiler#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Wenmj201582** [comment]
English：
We want to make a requirement in openeuler and understand that the bolt source code is now integrated into the llvm source code, which is a subdirectory under the llvm source code.
I'd like to ask whether bolt wants to build llvm IR from binary code and start the llvm back-end compilation function; Or how to integrate bolt and llvm in the later stage, and whether there are relevant plans?

中文：
我们想在openEuler做一个需求，了解到BOLT源代码现已合入LLVM源代码，是llvm源代码下的一个子目录。
想咨询一下BOLT是否要从二进制代码构建llvm IR，并启动llvm后端编译功能；或者说BOLT与llvm后期如何融合，是否有相关计划？

**Reply by arunkumarbhattar** [reply]
Greetings,
Unfortunately this is fork of checked-c clang compiler does not have OpenEuler or Bolt within its developmental scope as of this moment. 
Please forward your query to the original clang Github Repo --> https://github.com/llvm/llvm-project.
Thanks
